.github/CONTRIBUTING.md

```diff
@@ -164,6 +164,10 @@ CGO_ENABLED=1 GOBUILDFLAGS="-race" ./hack/test integration
 Set `TEST_KEEP_CACHE=1` for the test framework to keep external dependant images in a docker volume
 if you are repeatedly calling `./hack/test` script. This helps to avoid rate limiting on the remote registry side.
 
+You can also set `MOUNT_BUILDKIT_DOCKER_CONFIG_PATH` to forward docker config that will be used to pull
```

Comment:
Keep test env prefix consistent?

```suggestion
You can also set `BUILDKIT_TEST_MOUNT_DOCKER_CONFIG_PATH` to forward docker config that will be used to pull
```

Reply:
It's consistent with the parameters in `hack/shell`